addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -220,6 +232,8 @@ func _get_history(buffer: Dictionary, tick: int) -> Dictionary:
 	var latest = buffer.keys().max()
 
 	if tick < earliest:
+		_logger.warning("Tried to load tick %s which is earlier than the earliest we have recorded (%s)
+		Try increasing the history limit." % [tick, earliest])
```

Comment:
Debug comment, to confirm #260  is fixed. Place this message on main branch, and you will see it goes from 0 to current...